tests/components/main.test.tsx

```diff
@@ -8,14 +10,27 @@ jest.mock('next/router', () => ({
   useRouter: jest.fn(),
 }))
 
+let scenarioContent: ScenarioContent
+
+beforeAll(() => {
+  const pendingDeterminationScenario = { pendingDetermination: ['temporary text'] }
+  scenarioContent = getScenarioContent(pendingDeterminationScenario)
+})
```

Comment:
@lomky @kalvinwang  This chunk of code is reused in both `main.test.tsx` and `index.test.tsx`. Any ideas for where it could go to be DRY-er?

Reply:
If we want to do minimal worrying on refactoring here while we're focusing on business logic, I think my simplest suggestion for drying the tests up would be to just drop the redundant test! Once we have a stable first pass of business logic we can revisit testing the component layers and we'll have more perspective on where calls might go best.

Reply:
Sounds good. I'll do that for this PR. 

For our future conversation: After talking it through with @kalvinwang, I think I'm leaning towards keeping the `Main` component and moving the business logic into that layer.